message_definitions/v1.0/ras_a.xml

```diff
@@ -0,0 +1,124 @@
+      </description>
+      <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot).</field>
+      <field type="uint64_t" name="time_utc_detected" units="ms" invalid="0">Timestamp (time since UNIX epoch) of the POI detection, in UTC. 0 for unknown.</field>
+      <field type="uint64_t" name="time_utc_updated" units="ms" invalid="0">Timestamp (time since UNIX epoch) of the last POI update, in UTC. 0 for unknown.</field>
+      <field type="uint8_t" name="confidence_overall" units="%" invalid="UINT8_MAX">Generic confidence level. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint8_t" name="confidence_detection" units="%" invalid="UINT8_MAX">Confidence level of the POI detection. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint8_t" name="confidence_classification" units="%" invalid="UINT8_MAX">Confidence level of the POI classification. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint8_t" name="confidence_localization" units="%" invalid="UINT8_MAX">Confidence level of the POI localization. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint16_t" name="ttl" units="s">Time to live: If this time has elapsed, the POI should be deleted on the receiver side. A value of 0 should indicate no timeout.</field>
+      <field type="uint8_t" name="flags" display="bitmask">Bitmask for POI state. Bit 1: POI is in focus on camera, Bit 8: POI has been cleared and should be deleted</field>
+      <field type="uint64_t" name="uid" invalid="0">Unique ID for a given POI. Updates to a POIs information should use the same uid. 0 means unknown.</field>
+      <field type="int32_t" name="latitude" units="degE7" invalid="0">Latitude (WGS84) of the POI. If unknown: 0 (both Lat/Lon).</field>
+      <field type="int32_t" name="longitude" units="degE7" invalid="0">Longitude (WGS84) of the POI. If unknown: 0 (both Lat/Lon).</field>
+      <field type="int32_t" name="alt_amsl" units="mm" invalid="INT32_MAX">Altitude AMSL of the POI. Positive for up. INT32_MAX if unknown.</field>
+      <field type="int32_t" name="alt_elep" units="mm" invalid="INT32_MAX">Altitude above the EGM96 ellipsoid of the POI. Positive for up. INT32_MAX if unknown.</field>
+      <field type="uint32_t" name="classification" enum="POI_CLASSIFICATION">Classification of the POI. Can either used the POI_CLASSIFICATION enumeration (0x0 - 0x7FFFFFFF reserved), or use the reserved range for implementation specific (0x80000000 - UINT32_MAX).</field>
+      <field type="float" name="x" units="m" invalid="NaN">X position of the POI in the NED frame. NAN if unknown.</field>
```

Comment:
after talking with my team, I wonder if it makes since to include a frame field, with enum MAV_FRAME, to describe the frame used in the following fields

Reply:
Yes, if you want to be able to define POI in terms of vehicle-relative positioning/velocity at some point in future. Note however that this report both local and global frames so you need to think about what fields the frame applies to - for example if you want to support both vehicle  specific stuff and global things like terrain.

Reply:
we are mostly concerned with the fields that reference a NED frame. I think we are comfortable with the global fields remaining as well. We just aren't confident we are sure what is intended frame origin in this referenced NED frame

Reply:
Depends on how it is set - if this is from https://mavlink.io/en/messages/common.html#MAV_CMD_CAMERA_TRACK_POINT then best person to advise would be @julianoes .

Commonly the origin of NED frames is the EKF origin - the point at which the EKF estimator first got a valid position. Julian will be able to confirm if the POI is different - he has had lots to do with the gimbal/camera definitions.

As an aside, the usual pattern for position setting/getting messages like https://mavlink.io/en/messages/common.html#POSITION_TARGET_LOCAL_NED has been for there to be separate methods for local vs global, and that both messages include a frame. 
It's a bit abstract because most flight stacks only support one local frame. Further, there is no way to determine what local frames are supported by these kinds of messages (yet).

Reply:
@thomas-watters-skydio the local NED frame here would either be the local navigation frame of the drone or a common local frame of reference (in NED), i.e. common to multiple systems.

Reply:
MAV_FRAME will not anything more than complexity to this message, because as soon as you start adding more frames of reference, you multiply the number of cases that might have to be supported on both sides of the message stream. Rather simpler to consider that this is a local NED frame that matches a single or common navigation frame.